awx/ui_next/src/screens/Job/JobOutput/shared/jobOutputUtils.js

```diff
@@ -0,0 +1,29 @@
+export default function getRowRangePageSize(startIndex, stopIndex) {
+  let page;
+  let pageSize;
+
+  if (startIndex === stopIndex) {
+    page = startIndex;
```

Comment:
I think there's an off-by-one here.  This seems to fix it:
```suggestion
    page = startIndex + 1;
```

A reproducer for the bug is when `startIndex = 50` and `stopIndex = 50`:

![Screenshot from 2021-02-02 15-20-50](https://user-images.githubusercontent.com/9753817/106657917-9b37db80-656a-11eb-81ac-fc7c67a622f3.png)